### ISSUE TITLE
Can non-parameter wrappers be used?

### ISSUE THREAD

**Issue by dartiss** [wp-shortcake/shortcake#674]
I have a number of plugins that have shortcodes that have no parameters. Instead they wrap around, say, some text to apply a change to it. However, I can't see a way to achieve this as Shortcake only inserts the opening shortcode and doesn't allow for highlighting of content either (where it would then, ideally, put start and end shortcodes around the selected content).

I know it doesn't seem to add much benefit using Shortcake in this instance, as no parameters are required, the ease of looking up the functionality from an iconised menu, rather than remembering the shortcode name, does indeed provide a benefit that I'd, ideally, like to take advantage of.

Any thoughts on this will be appreciated.

**Comment by goldenapples** [comment]
No, there hasn't been any real progress on being able to use Shortcake views for wrapper shortcodes like that. I think it would be great to have a UX brainstorm to mock up some concepts of how that could look in the editor. As it stands, I don't have any concept of how rendering these types of shortcodes in the editor would work, other than possibly adding "start" and "end" markers and indenting the text inside of them.

**Comment by dartiss** [comment]
I'm happy to assist anyway I can.

It's possible now using the inner content to add something from Shortcake but what's really needed is an ability to highlight content before heading into Shortcake for this to be added in as a default value. The same with then editing - picking up the content for editing as part of the other shortcake fields is really needed. In my own experience, most shortcodes have start/end markers and content between.
